packages/cra-template/template.json

```
@@ -0,0 +1,19 @@
+{
+  "package": {
+    "dependencies": {
+      "@fluentui/react": "^7.155.3",
+      "@testing-library/jest-dom": "^5.11.4",
+      "@testing-library/react": "^11.1.0",
+      "@testing-library/user-event": "^12.1.10",
+      "@types/node": "^12.0.0",
+      "@types/react": "^17.0.0",
+      "@types/react-dom": "^17.0.0",
+      "@types/jest": "^26.0.15",
+      "typescript": "^4.1.2",
```

Might be just preference but I think it's safer to limit TS to a particular minor version, due to potential for significant changes between minor versions.
```suggestion
      "typescript": "~4.1.2",
```

---

that's what default CRA does. CRA has various anti patterns for whatever reasons ( putting everything to dependencies and various other things....). 

From my experience even using `~` to specify version is quite dangerous (also typescript semver is not a real semver) and exact versions should be used whenever possible.

---

Would you agree to chase it in a subsequent PR?

---

> From my experience even using `~` to specify version is quite dangerous (also typescript semver is not a real semver) and exact versions should be used whenever possible.

An exact TS version would be fine with me too.